catalog/task/create-internal-request/0.6/README.md

```diff
@@ -0,0 +1,34 @@
+# create-internal-request
+
+Creates an InternalRequest resource to call IIB service
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| pipelineRunName | The name of the Parent PipelineRun for this task | No | `ir-$(context.pipelineRun.name)` |
+| request | Internal pipeline request name | No | |
+| params | Internal Request parameters | No | |
+| inputDataFile | Optional file to read data from | Yes | "" |
+| requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | 360 |
+
+## Changelog
+
+### changes since 0.5
+- adds `extraData` result
```

Comment:
can you add some info about what this is about? I have no idea what to expect from this result looking at the README or code

Reply:
Also, can you show us the diff for the new version? It is visible for the fbc pipeline (since there is a separate commit for the copying), but not here.

Reply:
I renamed this to misc to not cause confusion with the extraData ticket. The think is that `create-internal-request` is a generic task that can be used by anything that needs to create internal requests. I have a new conditional that needs to be saved somewhere but can not have a specific name because this can be different for other use cases.

For this case I save the `fbc_opt_in=(true|false)` on it to know if I should sign and publish or not.